shared/src/address/payload.rs

```diff
@@ -19,6 +19,18 @@ pub struct DelegatedAddress {
     buffer: [u8; MAX_SUBADDRESS_LEN],
 }
 
+#[cfg(feature = "arb")]
+impl quickcheck::Arbitrary for DelegatedAddress {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        let buffer: [u8; MAX_SUBADDRESS_LEN] = core::array::from_fn(|_| u8::arbitrary(g));
+        Self {
+            namespace: ActorID::arbitrary(g),
+            length: usize::arbitrary(g),
```

Comment:
This should be bounded by `MAX_SUBADDRESS_LEN`.

Reply:
fixed